pennylane_lightning/lightning_qubit/lightning_qubit.py

```diff
@@ -1,4 +1,4 @@
-# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
```

Comment:
To make it short, imo all changes to this file should be reverted except for the year bump :sweat_smile: